camunda-formio-tasklist-vue/src/styles/camundaFormIOTasklist.scss

```diff
@@ -507,4 +510,7 @@ li {
   overflow: scroll !important;
   height: 93vh;
 }
+.text-font { 
```

Comment:
Can add text-font with a prefix like cft to avoid any potential css conflicts with integrated solution